tgui/packages/tgui/interfaces/common/AtmosControls.tsx

```diff
@@ -3,25 +3,46 @@ import { useBackend } from '../../backend';
 import { Button, LabeledList, NumberInput, Section } from '../../components';
 import { getGasLabel } from '../../constants';
 
-export const Vent = (props, context) => {
+export type VentData = {
+	id_tag: string;
```

Comment:
Mixing tabs and spaces, please run a linter with --fix. Or just fix your vscode setup (Ctrl+Shift+P, Typescript Version, choose 4.3.5-sdk), after that saving the file automatically fixes it.